src/plugins/timelist/Timelist.vue

```diff
@@ -357,21 +358,24 @@ export default {
             this.listActivities();
         },
         filterActivities(activity, index) {
-
             const hasFilterMatch = this.filterByName(activity.name);
 
             if (hasFilterMatch === false || this.hideAll === true) {
                 return false;
             }
 
-            if (this.showAll === true) {
+            if (this.isEditing && this.showAll) {
                 return true;
             }
 
             //current event or future start event or past end event
             const isCurrent = (this.noCurrent === false && this.timestamp >= activity.start && this.timestamp <= activity.end);
-            const isPast = (this.timestamp > activity.end && (this.viewBounds.pastEnd === undefined || activity.end >= this.viewBounds.pastEnd(this.timestamp)));
-            const isFuture = (this.timestamp < activity.start && (this.viewBounds.futureStart === undefined || activity.start <= this.viewBounds.futureStart(this.timestamp)));
+            const isPast = (this.timestamp > activity.end && (this.viewBounds?.pastEnd === undefined || activity.end >= this.viewBounds?.pastEnd(this.timestamp)));
+            const isFuture = (this.timestamp < activity.start && (this.viewBounds?.futureStart === undefined || activity.start <= this.viewBounds?.futureStart(this.timestamp)));
+
+            if (this.isFixedTime) {
+                return isCurrent || isFuture;
```

Comment:
See comment above about honoring the user's configuration for past/present/future when using fixed timespan mode.